src/components/date_picker/super_date_picker/quick_select_popover/_quick_select_popover.scss

```diff
@@ -23,9 +23,11 @@
 }
 
 // sass-lint:disable no-important
-.ouiQuickSelectPopover__buttonText {
-  // Override specificity from universal and sibling selectors
-  margin-right: $ouiSizeXS !important;
```

Comment:
I see that the new rule applies to elements with classname `ouiQuickSelectPopover__buttonText` when inside a prepend but it removes the rule that applied margin to all other elements with classname `ouiQuickSelectPopover__buttonText`. Shouldn't we retain that? Are we not breaking anything else by removing this?

Reply:
I did check it, every repo in the opensearch-project is using this class for date-picker. It is only dashboards which is using this class for the save button. 

Class references: https://github.com/search?q=org%3Aopensearch-project+euiQuickSelectPopover__buttonText&type=code

Reply:
Reverted to original class. Reasoning based on offline discussion with @AMoo-Miki  since, OUI is a styling library:
1. It is expected users can use anything/everything exposed by OUI from components to styling.
2. We shouldn’t break existing stuff for plugins/core or community partners